fusionauth/resource_fusionauth_application.go

```diff
@@ -686,10 +718,26 @@ func newOAuthConfiguration() *schema.Resource {
 				Default:     false,
 				Description: "When enabled the user will be required to be registered, or complete registration before redirecting to the configured callback in the authorization code grant or the implicit grant. This configuration does not currently apply to any other grant.",
 			},
-			"provided_scope_policy": {
-				Type:     schema.TypeList,
-				Optional: true,
-				Elem:     newOAuthConfigurationScopePolicy(),
+			"scope_handling_policy": {
+				Type:        schema.TypeString,
+				Optional:    true,
```

Comment:
Let's make this field required as discussed. The data migration set this to `Compatibility` to maintain backwards compatibility, but new applications set it to `Strict`.